src/Teams/Data/IRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+
+namespace Teams.Data
+{
+    interface IRepository<T> where T : class, IQueryable
```

Comment:
Please explain why we are using inheritance from IQueryable?

Reply:
Removed that, it is abundant.

Reply:
Do not reply to comment that the issue is fixed until it's committed